radixdlt/babylon-wallet-android
# Use the MainViewModel for onboarding flow

## Description
This PR fixes the onboarding flow that had one extra viewmodel by removing it and attach it on the MainVIewModel.
It also deletes the obsolete and unused WalletScreenNav. 


## How to test

1. Delete the wallet 
2. Start onboarding with account scan by using the seed phrase from [here](https://github.com/radixdlt/babylon-wallet-android/pull/693)
3. Once scan is complete you must see the "Recovery Complete" screen 

Try also to navigate back and forth in onboarding flow or to exit app and launch it again.

## Video
https://github.com/radixdlt/babylon-wallet-android/assets/118305718/145f1093-37ad-4b50-924a-c4e087cf763b



## PR submission checklist
- [X] I have tested onboarding flow with different scenarios
